# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: D:\Zend\ZendServer\share\ZendFramework\trunk\library\Zend\Dojo\View\Helper
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Slider.php
--- Slider.php Base (BASE)
+++ Slider.php Locally Modified (Based On LOCAL)
@@ -77,8 +77,14 @@
         }
 
         $content = '';
-        $params['value'] = $value;
 
+        // check required for IE
+        if ($value === null) {
+                $param['value'] =  $param['minimum'];
+        } else {
+                $param['value'] =  $value;
+        }
+
         if (!array_key_exists('onChange', $attribs)) {
             $attribs['onChange'] = "dojo.byId('" . $id . "').value = arguments[0];";
         }
